way_to_home/custom_user/models.py

```diff
@@ -27,6 +28,30 @@ def to_dict(self):
             'phone_number': self.phone_number
         }
 
+    def update(self, password=None, google_token=None, phone_number=None, is_active=None):
+        """Method for object update."""
+        if password:
+            self.password = password
+        if google_token:
+            self.google_token = google_token
+        if phone_number:
+            self.phone_number = phone_number
+        if is_active is not None:
+            self.is_active = is_active
+        try:
+            self.save()
+            return True
+        except ValueError:
+            return False
+
+    @classmethod
+    def get_by_email(cls, email):
+        """Method for returns user by email"""
+        try:
+            return cls.objects.get(email=email)
+        except (ValueError, IntegrityError):
```

Comment:
```suggestion
        except (ValueError, IntegrityError):
```
? I think the exception is only `cls.DoesNotExist`

Reply:
done